locale/de.json

```diff
@@ -422,6 +422,7 @@
     "Ordinary":"Standardform",
     "Origin":"Ur",
     "Overcast":"Wolken",
+    "overview":"übersicht",
```

Comment:
Really lowercased?

Reply:
Dosent matter in commands will you prefer uppercase?

Reply:
Oh sure, you are right